packages/css/src/image/README.md

```diff
@@ -0,0 +1,16 @@
+# Image
+
+Toont een afbeelding.
+
+## Richtlijnen
+
+- Vergeet niet om een beschrijving van de afbeelding op te nemen in het `alt`-attribuut.
+  Dit zorgt ervoor dat gebruikers van schermlezers deze informatie ook tot zich kunnen nemen.
+  Daarnaast kan het helpen bij zoekmachineoptimalisatie.
+- Alleen voor decoratieve afbeeldingen is zo’n beschrijving niet nodig.
+  Denk aan afbeeldingen die weinig toevoegen aan de nabije tekst of afbeeldingen die louter bijdragen aan het ontwerp of de sfeer van de pagina (bron: [W3C Web Accessibility Initiative](https://www.w3.org/WAI/tutorials/images/decorative/)).
+- Zorg ervoor dat de afbeelding een beeldverhouding heeft die ondersteund wordt door het [Aspect Ratio](?path=/docs/layout-aspect-ratio--docs) component.
+
+## Relevante WCAG-eisen
+
+- [WCAG 1.1.1](https://www.w3.org/TR/WCAG22/#non-text-content): niet-tekstuele content heeft een tekstueel alternatief
```

Comment:
Do we want to add [1.4.5 and 1.4.9](https://www.w3.org/WAI/tutorials/images/#related) here as well? We should at least mention that you shouldn't use images of text if at all avoidable.

Reply:
Yes, fine, a was already wondering how we decided on the right combination of guidelines – check all of them individually whether they apply or find guidance on grouping like this. I’ve added those.

We might want to create some handy construct to reuse the content of each guidelines.

Reply:
Which guidelines we add is currently a bit ad hoc I think, but that's not necessarily a problem imo. As long as we add new guidelines in the right place(s) when we run into them.